bip-0047.mediawiki

```diff
@@ -365,6 +365,37 @@ Bob detects notification transactions by adding his payment code identifier to h
 
 Alice's wallet should spend the notification change output at the next appropriate opportunity.
 
+==Version 3==
+
+A user of version 3 payment codes may use the same payment code to receive payments on different blockchains without address collisions.
+
+Version 3 payment codes behave identifically to version 2 payment codes, except as modified below.
+
+===Representation===
+
+====Binary Serialization====
+
+* Byte 0: version. required value: 0x03
+
+===Protocol===
+
+====Sending====
+
+
+# Each time Alice wants to initiate a transaction to Bob, Alice derives a unique P2PKH address for the transaction using ECDH as follows:
+## Alice selects the 0th private key derived from her payment code: <pre>a</pre>
+## Alice selects the next unused public key derived from Bob's payment code, starting from zero: <pre>B, where B = bG</pre>
+### The "next unused" public key is based on an index specific to the Alice-Bob context, not global to either Alice or Bob
+## Alice selects the 4 byte registered coin type for the transaction based on SLIP-0044: <pre>t</pre>
```

Comment:
N.B.: The coin types accepted by Bob for payment will be need to negotiated at another protocol layer since SLIP-0044 has many coins and is subject to updates.

Reply:
Addressed now with a comment at the beginning of the sending procedure.

Reply:
ACK